docs/user/openstack/known-issues.md

```diff
@@ -87,6 +87,15 @@ volumeBindingMode: WaitForFirstConsumer
 
 [nova-az-setting]: ../openstack#setting-nova-availability-zones
 
+## Problems when changing the Machine spec for master node
+
+Changing the Machine spec (e.g. the image name) for master node will delete the instance in OpenStack and the machine status will change to `phase: failed with errorMessage: Can't find created instance`.
+This is a known limitation in CAPO where we don't support that action.
```

Comment:
Please avoid or unroll acronyms, as this is customer-facing documentation (or the source thereof).